### ISSUE TITLE
I'm sorry to bother you,but I have some questions...

### ISSUE THREAD

**Issue by erhuodaosi** [pkuCactus/BDCN#11]
@pkuCactus 

Hi,

What an awesome job! I have some questions in the sketch of the places2 dataset(256 * 256).How should I modify the parameter of yita in config.py?Could it work on the places2 dataset?How long does it cost if I want to get the edge flist form the whole places2 dataset?

I would very appreciate if you could help me,thank you!

Thanks,

**Comment by pkuvmc** [comment]
If your edges get from segmentation, you need not care about the parameter yita. If you edges is  probability in (0,1), the yita control the confidence that the edges of human consistency, so you can set whatever more than 0, mostly is 0.5, which means more than half of annotators agree that the edges are edges.

**Comment by pkuCactus** [comment]
If your edges get from segmentation, you need not care about the parameter yita. If you edges is probability in (0,1), the yita control the confidence that the edges of human consistency, so you can set whatever more than 0, mostly is 0.5, which means more than half of annotators agree that the edges are edges.

**Comment by erhuodaosi** [comment]
@pkuCactus

Hi,

Thank you for your timely reply!Could I get the edge flist without training the model or just use the pretrained model you provide to get the edge flist from the whole Places2 dataset?

I would very appreciate if you could help me,thank you!

Thanks,

**Comment by pkuCactus** [comment]
Yes, you can use the pretrained model to get the edge while it may not very good but indeed not bad because the detector has a pretty generalization. The better way is you finetune on the dataset you used.

**Comment by erhuodaosi** [comment]
@pkuCactus

Hi,

Thank you for your timely response!I would have a try,thank you!

Thanks,

**Comment by caihongzhiyue01** [comment]
> @pkuCactus
> 
> Hi,
> 
> What an awesome job! I have some questions in the sketch of the places2 dataset(256 * 256).How should I modify the parameter of yita in config.py?Could it work on the places2 dataset?How long does it cost if I want to get the edge flist form the whole places2 dataset?
> 
> I would very appreciate if you could help me,thank you!
> 
> Thanks,



> @pkuCactus
> 
> Hi,
> 
> What an awesome job! I have some questions in the sketch of the places2 dataset(256 * 256).How should I modify the parameter of yita in config.py?Could it work on the places2 dataset?How long does it cost if I want to get the edge flist form the whole places2 dataset?
> 
> I would very appreciate if you could help me,thank you!
> 
> Thanks,



> @pkuCactus
> 
> Hi,
> 
> Thank you for your timely reply!Could I get the edge flist without training the model or just use the pretrained model you provide to get the edge flist from the whole Places2 dataset?
> 
> I would very appreciate if you could help me,thank you!
> 
> Thanks,

Hello, your idea is very good and I also want to use the pretrained model for obtaining the edge on other datasets. Do you finish your idea?

**Comment by caihongzhiyue01** [comment]
> @pkuCactus
> 
> Hi,
> 
> What an awesome job! I have some questions in the sketch of the places2 dataset(256 * 256).How should I modify the parameter of yita in config.py?Could it work on the places2 dataset?How long does it cost if I want to get the edge flist form the whole places2 dataset?
> 
> I would very appreciate if you could help me,thank you!
> 
> Thanks,

Hello, your idea is very good and I also want to use the pretrained model for obtaining the edge on other datasets. Do you finish your idea?
